MANIFEST.in

```diff
@@ -3,3 +3,4 @@ include conbench.png
 include requirements-build.txt
 include requirements-cli.txt
 include requirements-test.txt
+include version.py
```

Comment:
Does this need to be in the manifest?

`include version.py`

Reply:
Yep. This file is used by `setup.py`

I added version.py so I can do this `version=$(cat version.py | tr -d '"' | tr -d '__version__ = ')`